feature/home/src/main/java/io/filmtime/feature/home/HomeScreen.kt

```diff
@@ -34,11 +50,46 @@ fun HomeScreen(
   onMovieClick: (tmdbId: Int) -> Unit,
   onShowClick: (tmdbId: Int) -> Unit,
 ) {
+
+
   val state by viewModel.state.collectAsStateWithLifecycle()
+  var load by remember { mutableStateOf(false) }
+
+
+  LaunchedEffect(load) {
+    if (load) {
+      viewModel.retry()
+      viewModel.loadTrendingMovies()
+      viewModel.loadTrendingShows()
+      load = false
+
+    }
+  }
```

Comment:
Why not move this logic to the ViewModel? Since we already have a ViewModel, it would be better to centralize all responsibilities there.

in viewmodel retry function:

```kotlin
fun retry() {
 // reset error state
 // call loadTrendingMovies
 // call loadTrendingShows
}
```